src/main/java/com/jalasoft/webservice/controller/DownloadController.java

```diff
@@ -69,4 +69,26 @@ public void getFile(HttpServletResponse response,
             LOGGER.info("File was not download..." + e.getMessage());
         }
     }
+
+    /***
+     *
+     * @param response
+     * @param fileName  File Name to download
+     */
+    @GetMapping("/filemetadatazip/{fileName:.+}")
+    public void getFileAndMetadataInfo(HttpServletResponse response,
+                                       @PathVariable("fileName") String fileName) {
+        String fullPathName = String.format("%s%s", PropertiesManager.getInstance().getPropertiesReader().getValue(targetFileKey), fileName);
+        String commonContentType = "application/octet-stream";
+        try {
+            File file = new File(fullPathName);
+            if (file.exists()) {
+                response.setContentType(commonContentType);    // Download the file directly
+                InputStream is = new BufferedInputStream(new FileInputStream(fullPathName));
+                FileCopyUtils.copy(is, response.getOutputStream());
+            }
+        } catch (IOException ex) {
+            ex.printStackTrace();
```

Comment:
Use LOGGER to print the error. Also don't you think we need to return a Response error?